kafo_parsers.gemspec

```diff
@@ -23,6 +23,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "ci_reporter", "~> 1.9.0"
 
   # puppet manifests parsing
-  spec.add_dependency 'puppet', '< 4.0.0' 
-  spec.add_dependency 'rdoc', '>= 3.9.0'
+  spec.add_development_dependency 'puppet', '< 4.0.0'
+  spec.add_development_dependency 'rdoc', '>= 3.9.0'
```

Comment:
I think this is still required by [doc_parser](https://github.com/theforeman/kafo_parsers/blob/master/lib/kafo_parsers/doc_parser.rb#L2) and I think other parses would need it too, except the cache one which is in core. If you think this should be optional then I think [doc_parser requirement](https://github.com/theforeman/kafo_parsers/pull/14/files#diff-6c8941233b68b6e03805ab782dd6f416R2) should be moved to `available?` as well